flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["i386", "x86_64"]
```

Comment:
What would it take to fix the ARM builds? Or is there some fundamental incompatibility for OpenSCAD on ARM? It would be nice to be able to keep running this app on ARM. (Otherwise, if the existing ARM build does not actually run, feel free to disregard this comment!)

Reply:
OpenSCAD doesn't support OpenGL ES: https://github.com/flathub/org.openscad.OpenSCAD/pull/6#issuecomment-559858247